doc/ontodoc/mkdoc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Bash script for generating documentation
+set -e
+set -x
+
+# Directories
+rootdir=$(git rev-parse --show-toplevel)
+ontodocdir=$rootdir/doc/ontodoc
+tmpdir=$ontodocdir/tmp
```

Comment:
Not a crucial changes. Just there to help the eye and the interpreter.
```suggestion
ontodocdir=${rootdir}/doc/ontodoc
tmpdir=${ontodocdir}/tmp
```